.github/workflows/load-env/action.yml

```diff
@@ -0,0 +1,35 @@
+name: "Load .env Variables"
+description: "Loads environment variables from a .env file into GITHUB_ENV"
+author: "ChatGPT"
+
+inputs:
+  env-file-path:
+    description: "Path to the .env file"
+    required: true
+    default: "gha.env"
+
+runs:
+  using: "composite"
+  steps:
+    - name: Read .env File and Export Variables
+      shell: bash
+      run: |
+        set -a
+        # place repo name and owner into ENV for the image name
+        echo "REPOSITORY_NAME=${{ github.event.repository.name }}" >> $GITHUB_ENV
```

Comment:
GitHub repos can use underscores, so want to be careful here. Granted, most cases will be folks simply forking this though.